### PR TITLE
Typographical fixes

### PR DESCRIPTION
**Pull request by ldiringer**

### CODE REVIEW

File: src/docs/resources/javascripts/tabcontainer.js [hunk 1]
```diff
@@ -22,15 +22,15 @@
 			if(contentHeight>height) {
 				height = contentHeight;
 			}
-			$(tab).prepend('<input type="radio" id="tab-'+group+'-'+i+'" name="tab-group-'+group+'" '+(i==0 ? 'checked' : '')+'>');
+			$(tab).prepend('<input type="radio" id="tab-'+group+'-'+i+'" name="tab-group-'+group+'" '+(i==0 ? 'checked': '')+'>');
```
Comment (aurelien-baudet) [inline src/docs/resources/javascripts/tabcontainer.js]: Not typo in code ;)

File: src/docs/resources/javascripts/tabcontainer.js [hunk 2]
```diff
@@ -22,15 +22,15 @@
...
 			$(tab).append('<div class="tab-content">');
 			var tabContentNode = $(tab).find('.tab-content');
 			$(tabContentNode).append(contentNodes);
 			$(tab).append(tabContentNode);
 			$(start).append(tab);
 			tabTitleNode.replaceWith('<label class="tab-label" for="tab-'+group+'-'+i+'">'+tabTitle+'</label>');
 		}
-		$(start).css('height', (height+45+42)+'px');			// TODO: height of tab... This is really bad :(
+		$(start).css('height', (height+45+42)+'px');			// TODO: height of tab... This is really bad:(
```
Comment (aurelien-baudet) [inline src/docs/resources/javascripts/tabcontainer.js]: Not typo in code ;)

File: src/docs/resources/javascripts/steps.js [hunk 3]
```diff
@@ -17,7 +17,7 @@
 		parent.find('.progressbar').append(parent.find('p'));
 		for(var i=0 ; i<steps.length ; i++) {
 			var step = $(steps[i]);
-			step.addClass('step step'+i+' '+(current==i ? 'current' : '')+' '+(i==steps.length-1 ? 'last' : ''));
+			step.addClass('step step'+i+' '+(current==i ? 'current': '')+' '+(i==steps.length-1 ? 'last': ''));
```
Comment (aurelien-baudet) [inline src/docs/resources/javascripts/steps.js]: Not typo in code ;)

File: src/docs/resources/javascripts/sourcecode/source-utils.js [hunk 4]
```diff
@@ -8,8 +8,8 @@
 			if(part.indexOf("-")!=-1) {
 				// example: 10-25
 				var subpart = part.split("-");
-				var start = subpart[0] ? parseInt(subpart[0]) : 1;
-				var end = subpart[1] ? parseInt(subpart[1]) : sourceLines.length;
+				var start = subpart[0] ? parseInt(subpart[0]): 1;
```
Comment (aurelien-baudet) [inline src/docs/resources/javascripts/sourcecode/source-utils.js]: Not typo in code ;)

File: src/docs/resources/javascripts/sourcecode/highlight-lines.js [hunk 5]
```diff
@@ -66,7 +66,7 @@
 	};
 
 	var decorate = function (/*String*/line, /*String*/type) {
-		return '<span class="highlight-line' + (type ? '-' + type : '') + '"><span class="wrapped-line">' + (line || "") + '</span></span>';
+		return '<span class="highlight-line' + (type ? '-' + type: '') + '"><span class="wrapped-line">' + (line || "") + '</span></span>';
```
Comment (aurelien-baudet) [inline src/docs/resources/javascripts/sourcecode/highlight-lines.js]: Not typo in code ;)

**Comment by ldiringer** [comment]
Thank you @aurelien-baudet, I've removed changes from .js files.

**Pull request merged**